test/ell1.d

```diff
@@ -11,3 +12,37 @@ unittest
             .shouldFailWith("left recursion");
     }
 }
+
+@("issue #11: disallow trailing content before EOF at LL(1) parser, by default")
+unittest
+{
+    with (sandbox)
+    {
+        const eag = `
+            NoEOF<+ 'Done': Done>: 'x'.
+            Done = 'Done'.
+            `.outdent;
+
+        run!"cat <<EOF | ./gamma --output-directory %s%sEOF"(directory, eag)
+            .shouldPassWith("NoEOF grammar is SLAG");
+        run!"cd %s && echo x yy zzz | ./NoEOF"(directory)
+            .shouldFailWith("syntax error, unexpected token\\(s\\) before end of file");
+    }
+}
+
+@("issue #11: allow trailing content before EOF at LL(1) parser if option -t has been passeed")
+unittest
+{
+    with (sandbox)
+    {
+        const eag = `
+            NoEOF<+ 'Done': Done>: 'x'.
+            Done = 'Done'.
+            `.outdent;
+
+        run!"cat <<EOF | ./gamma --output-directory %s%sEOF"(directory, eag)
+            .shouldPassWith("NoEOF grammar is SLAG");
+        run!"cd %s && echo x yy zzz | ./NoEOF -t"(directory)
```

Comment:
`-t` is for manual input; a script should use the long option `--tralingTokens`

Reply:
Yes, better readability in the tests.

Reply:
Not relevant anymore, we go for a backward incompatible change. See below.
This test will disappear either.